storage/migrations/02_oasis_3_runtimes.up.sql

```diff
@@ -98,6 +98,8 @@ CREATE TABLE oasis_3.runtime_events
   evm_log_params JSONB,
   related_accounts TEXT[]
 );
+CREATE INDEX ix_runtime_events_round ON oasis_3.runtime_events(runtime, round);  -- for sorting by round, when there are no filters applied
+CREATE INDEX ix_runtime_events_tx_hash ON oasis_3.runtime_events(tx_hash);
```

Comment:
Let's say mindful instead of worried:)
These are all indexes we need, there's not a clear alternative, but we should avoid introducing wasteful indexes.
They're actually also not problematic as long as we're doing one block every six second. The fast reindexing OTOH does suffer from all these indexes. My plan there is to have a fast analyzer mode where we disable/remove most indexes (and disable the API), and then enable them again at the end.